package.json

```diff
@@ -266,9 +266,9 @@
         "xmlbuilder2": "~3.0.2"
     },
     "devDependencies": {
+        "@faker-js/faker": "^6.0.0-alpha.7",
```

Comment:
Can we use a stable version, like 5.5.3?

Reply:
It seems that version 5.5.3 doesn't have [TypeScript Support](https://github.com/faker-js/faker#typescript-support)